src/priority/priority.py

```diff
@@ -295,6 +295,11 @@ def __init__(self, maximum_streams=1000):
         self._root_stream = Stream(stream_id=0, weight=1)
         self._root_stream.active = False
         self._streams = {0: self._root_stream}
+
+        if not isinstance(maximum_streams, int):
```

Comment:
Meh, it’s an edge case, and “weird edge cases in Python 2” is near the bottom of the list of things I care about. 😉